### ISSUE TITLE
Need to redefine the equation of Impulse

### ISSUE THREAD

**Issue by oluowoj** [JacquesCarette/Drasil#1146]
Equation of impulse in manual version is shown below:
<img width="363" alt="image" src="https://user-images.githubusercontent.com/43192745/53594515-03a10580-3b69-11e9-96d6-e23c8399d907.png">

@JacquesCarette will discuss how to define the equation without 2 =

**Comment by smiths** [comment]
I don't know if it addresses the concerns of @JacquesCarette, but I think what we want to do is write the final form of the equation that we need and then use the derivation section to show the progression from the general equation to the specific equation.  In looking at the impulse equation now, there are definitely problems with it.  J is shown bold, as a vector, by m*Delta v is a scalar.  How did that happen?  There are steps missing.  My guess is that we want the scalar J = m * Delta v and we can show a way to derive this from the more general integral form of the impulse equation.

We are also making a transition from the continuous form of the equation to a discrete form.

We'll need to make sure that we have assumptions in place that let us get from the general form to the specific.

**Comment by JacquesCarette** [comment]
I agree that the equation should be written in its final form. And, for now, the derivation can show the rest. We'll want to 'upgrade' that later.

**Comment by oluowoj** [comment]
Impulse(vector) equation has been resolved and updated to
J = m*Delta v. Screenshot of .html output

<img width="340" alt="image" src="https://user-images.githubusercontent.com/43192745/53836241-780ae880-3f5d-11e9-976c-116e5222439b.png">

**Comment by smiths** [comment]
@oluowoj, this final equation looks good to me.  What branch is this in?  I went to have a look at the current documentation, but I want to make sure I'm in the right branch.  Is it gamephy_SRS?

**Comment by oluowoj** [comment]
yes, that's correct.

**Comment by smiths** [comment]
Thank you @oluowoj.  I had a quick look at the new documentation.  It looks to be going in the right direction to me.  I think we are safe to close this issue now.

**Comment by JacquesCarette** [comment]
I re-opened the issue because while the equation is now correct, the fix loses information.

**Comment by Awurama-N** [comment]
I was looking for the pull request that these changes were suppose to be made in and i found PR #1376 , I saw that you brought up the fact that things were going missing in the comment and the PR was closed and never merged. But I went through the changes made and compared it to the current state of the files that were suppose to be changed, I see some of the changes might have been made in other PR's since they exist now but others don't. 
Would you like me to go through any differences and replicate some of the changes to resolve this issue without any other information disappearing.

**Comment by JacquesCarette** [comment]
Yes please.
